### PR TITLE
Some reorganization of Attribute

### PR DESCRIPTION
**Pull request by VisLab**

### CODE REVIEW

File: HED-schema-reduced.mediawiki [hunk 1]
```diff
@@ -340,29 +350,55 @@ There are four ways to define the scope of an event.  The most common method is
 ** Urban
 ** Rural
 
-* Cognitive modifier <nowiki>{extensionAllowed}</nowiki>
-** Imagined <nowiki>[This is used to identity that the (sub)event only happened in the imagination of the participant, e.g.  imagined movements in motor imagery paradigms.]</nowiki>
-** Reflexive
-** Intentional
+* Experimental <nowiki>{extensionAllowed}</nowiki>
```
Comment (dungscout96) [inline HED-schema-reduced.mediawiki:353]: Still not sure if this the right term. Action-related modifiers can be used for experimental or non-task related action for example.

File: HED-schema-reduced.mediawiki [hunk 2]
```diff
@@ -340,29 +350,55 @@ There are four ways to define the scope of an event.  The most common method is
...
+****Negative valence 
+*** Priming 
+**** Motoric priming
+**** Emotional priming
+**** Perceptual priming
+*** Subliminal 
+**** Unmasked subliminal
+**** Masked subliminal
+**** Supraliminal <nowiki>[By default this is assumed about each stimulus]</nowiki> 
+**** Liminal <nowiki>At the 75%-25% perception threshold]</nowiki>
```
Comment (dungscout96) [inline HED-schema-reduced.mediawiki:369]: Aren't these Paradigm-related? These are duplicated below as well

File: HED-schema-reduced.mediawiki [hunk 3]
```diff
@@ -340,29 +350,55 @@ There are four ways to define the scope of an event.  The most common method is
 ** Urban
 ** Rural
 
-* Cognitive modifier <nowiki>{extensionAllowed}</nowiki>
-** Imagined <nowiki>[This is used to identity that the (sub)event only happened in the imagination of the participant, e.g.  imagined movements in motor imagery paradigms.]</nowiki>
-** Reflexive
-** Intentional
+* Experimental <nowiki>{extensionAllowed}</nowiki>
 ** Incidental
```
Comment (dungscout96) [inline HED-schema-reduced.mediawiki:354]: Can it be used anywhere else besides with Action?

Comment (VisLab) [reply HED-schema-reduced.mediawiki:354]: I agree it isn't quite there --- but I don't know how to fix.  Do you have any specific ideas?  If not, I may go ahead and merge so we can look again in the expanded view.  We need to a catch-all category.

**Pull request merged**